### DISCUSSION TITLE
insertPos

### DISCUSSION THREAD

**Discussion by pdavidow** [codedownio/myers-diff#9] [General]
```haskell
data Edit = EditDelete { deleteFrom :: Int
                       , deleteTo :: Int }
          | EditInsert { insertPos :: Int
                       , insertFrom :: Int
                       , insertTo :: Int }
 ```
What is the meaning of `insertPos` ?

**Comment by thomasjm** [comment]
I believe `insertPos` is the position of the insert (in the left string), and `insertFrom`/`insertTo` are the bounds of the inserted tokens (in the right string).

See the usage here:

https://github.com/codedownio/myers-diff/blob/600d2c4614266e8412cd3aa64e08f47b7060d4c2/src/Data/Diff/Myers.hs#L210

**Reply by pdavidow** [reply]
And for delete, the from/to is for the left string?

**Reply by thomasjm** [reply]
Yep
